docs/manuals/core/entities/data-node-mgt.md

```diff
@@ -933,10 +933,12 @@ To get the parent entities of a data node (scenarios or pipelines or tasks) you
     data_node = scenario.sales_history
 
     # Retrieve the parent entities of the data node
-    parent_entities = data_node.get_parents()   # {'scenarios': [Scenario 1], 'pipelines': [Pipeline 1], 'tasks': [Task 1]}
+    parent_entities = data_node.get_parents()
+    # {'scenarios': [Scenario 1], 'pipelines': [Pipeline 1], 'tasks': [Task 1]}
```

Comment:
May be
\# => {'scenarios': [Scenario 1], 'pipelines': [Pipeline 1], 'tasks': [Task 1]}
will be more readable....